index.org

```diff
@@ -103,51 +103,58 @@ novice]] and [[http://howardism.org/Technical/Emacs/literate-devops.html][powerf
 
 * Further reading
 
-As well at the [[file:quickstart.org][quickstart]] and [[file:manual/index.html][manual]], there's also
-a [[file:guide/index.html][guide]], and a wealth of community-written docs on [[https://orgmode.org/worg/][Worg]] .
+Beyond the [[file:quickstart.org][quickstart]] and the [[file:manual/index.html][manual]],
+Org has a [[file:guide/index.html][guide]], and a wealth of community-written documentation on [[https://orgmode.org/worg/][Worg]] .
 
-Once you've used Org for a bit, you'll know it's great --- it's actually
-so great that there have been [[https://orgmode.org/worg/org-papers.html][(scientific) papers]] written about the value of Org
-for [[https://en.wikipedia.org/wiki/Reproducibility#Reproducible_research][reproducible research]].
+Nearly every Org user has a story to tell about how Org enables and empowers them ---
+some have found it so useful that they have written [[https://orgmode.org/worg/org-papers.html][(scientific) papers]] about
```

Comment:
As above re: "story to tell", perhaps we could do the same with "they have written" - or maybe I'm overeager.

Reply:
Since this is already in Worg I think we can leave it as is. As a stop gap for the gallery, maybe it could be implemented as part of Worg so that the links stay inside the orgmode site?

Reply:
This does seem less significant, perhaps we should just leave this be at least for now.